rest/sip-in/delete-credential/delete-credential.4.x.rb

```diff
@@ -8,4 +8,4 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 @credential = @client.account.sip.credential_lists.get("CL32a3c49700934481addd5ce1659f04d2").credentials.get("SC32a3c49700934481addd5ce1659f04d2")
```

Comment:
This line is too long. What do you think about splitting it into several lines? - you can find more info about the recommended practices at https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains. I found **Option A** more intuitive.

```ruby
@credential = @client.account.sip
  .credential_lists
  .get("CL32a3c49700934481addd5ce1659f04d2")
  .credentials
  .get("SC32a3c49700934481addd5ce1659f04d2")
```